invoice/src/main/java/org/killbill/billing/invoice/api/svcs/DefaultInvoiceInternalApi.java

```diff
@@ -90,7 +90,7 @@ private DefaultInvoice getInvoiceByIdInternal(final UUID invoiceId, final Intern
 
     @Override
     public Collection<Invoice> getUnpaidInvoicesByAccountId(final UUID accountId, final LocalDate upToDate, final InternalTenantContext context) {
-        final List<InvoiceModelDao> unpaidInvoicesByAccountId = dao.getUnpaidInvoicesByAccountId(accountId, null, upToDate, context);
```

Comment:
yes that makes sense - you can remove he comment: This 'internal' api is used internally and since we never use the tracking ids internally, it should be safe to pass `false` for all methods in this class.